terraform/e2e/eks/main.tf

```
@@ -0,0 +1,128 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+module "common" {
+  source = "../../common"
+}
+
+module "basic_components" {
+  source = "../../basic_components"
+}
+
+locals {
+  aws_eks      = "aws eks --region ${var.region}"
```

--region is redundant since we already have --region in provider.tf
```suggestion
  aws_eks      = "aws eks"
```

---

That's the flag to pass the region in.

---

Will remove.